docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: '3'
+services:
+  app:
+    container_name: review
+    restart: always
+    build: .
+    ports:
+      - '80:3003'
+    links:
+      - mongo
```

Comment:
This is referencing a container you have not built yet, so it's a good idea to remove it for now. 

Also, the containers automatically link by container name.  

Also, this command is not recommended. See: https://docs.docker.com/network/links/